python/ray/autoscaler/aws/config.py

```diff
@@ -694,6 +725,136 @@ def _get_key(key_name, config):
         raise exc
 
 
+def _configure_from_launch_template(config):
+    for _, config_key in NODE_TYPE_CONFIG_KEYS.items():
```

Comment:
Minor: Since we're only using the values, should we just iterate over values instead of items?

```python
    for config_key in NODE_TYPE_CONFIG_KEYS.values():
        config = _configure_node_type_from_launch_template(config, config_key)
    return config
```